docs/source/formulations_example.rst

```diff
@@ -0,0 +1,648 @@
+     - Simple Syrup A Quantity
+     - Simple Syrup B Quantity
+     - Sugar Quantity
+     - Lime Juice Quantity
+     - Triple Sec Quantity
+     - Tequila Quantity
+     - Ice Quantity
+   * - Margarita A
+     - 6.3 +/- 0.1
+     - 8.0
+     - 0.20
+     - 0.0
+     - 0.0
+     - 0.15
+     - 0.0
+     - 0.25
+     - 0.40
+   * - Margarita B
+     - 5.4 +/- 0.1
+     - 12.0
+     - 0.0
+     - 0.15
+     - 0.0
+     - 0.10
+     - 0.10
+     - 0.30
+     - 0.35
+
+Ingesting Data
+--------------
+
+.. Warning:: Ingesting data to GEMD is best done with additional tooling to automate the process.
+
+
+Although we will not describe every step of the data ingestion process, we will highlight several data objects and their inter-connections.
+Perhaps most crucial are the `attribute templates` that correspond to the parameters of the blending process and the properties of the ingredients/formulations.
+The name, bounds, and units on these templates will later be matched to descriptors in the AI Engine.
+Make sure to set the bounds wide enough to encompass all anticipated use cases of the templates.
+
+.. note:: In the code examples that follow, imports are assumed to be cumulative.
+    Once an import has been demonstrated, we do not include it in subsequent code blocks.
+
+.. code-block:: python
+
+    from citrine.resources.parameter_template import ParameterTemplate
+    from citrine.resources.property_template import PropertyTemplate
+    from gemd.entity.bounds import RealBounds
+
+    # Assume we have already created a project and dataset, named "project" and "dataset."
+    # Though not demonstrated here, it is useful to give templates custom ids so that they can be easily retrieved.
+    blend_time_template = dataset.parameter_templates.register(
+        ParameterTemplate("blend time", bounds=RealBounds(0, 60.0, "s"))
+    )
+    price_template = dataset.property_templates.register(
+        PropertyTemplate("price", bounds=RealBounds(0, 100, "1/kg"))
+    )
+    sucrose_fraction_template = dataset.property_templates.register(
+        PropertyTemplate("sucrose fraction", bounds=RealBounds(0, 1, ""))
+    )
+    tastiness_template = dataset.property_templates.register(
+        PropertyTemplate("tastiness", bounds=RealBounds(0, 10, ""))
+    )
+
+The attribute templates are attached to relevant object templates.
+For example, the process template to represent blending should include ``blend_time_template`` as a ``parameter``.
+
+Although it contains no attributes, we should particular pay attention to the templates that represent the mixing processes.
+These will be used to convert GEMD data into formulations.
+It is possible to use different process templates to distinguish between different types of mixing, but here we use the one template to represesnt both types of mixing that occur (mixing the simple syrup and mixing the margarita ingredients).
+The template includes a comprehensive list of all allowed names and labels.
```

Comment:
SHOULD - Clarify that we mean _ingredient_ names here. Maybe:
> ...includes a comprehensive list of all allowed ingredient names and ingredient labels that can go into the process.

Often it's not possible to enumerate all the ingredient names/labels when making process templates. It may be good to call out that this is not a requirement for GEMD, but will make creating tables later on much easier.